charts/woodpecker/README.md

```diff
@@ -0,0 +1,49 @@
+# Woodpecker Helm Charts
+
+This repository contains the helm charts of the [Woodpecker](https://woodpecker-ci.org) project.
```

Comment:
Not sure if this is the expected content.

Reply:
Agreed, removed it. It's present in the top-level README anyhow.